metricbeat/etc/beat.yml

```diff
@@ -55,4 +55,5 @@ metricbeat.modules:
   period: 2s
   processes: ['.*']
 
-
+  # if true, exports the CPU usage in ticks, together with the percentage values
+  cpu_usage_in_ticks: false
```

Comment:
It seems like you updated this here directly? This should be updated in the system module and then you have to run `make collect`.